bundles/admin/admin-layereditor/view/AdminLayerForm/AdminLayerFormHandler.js

```diff
@@ -831,6 +833,20 @@ class UIHandler extends StateHandler {
         return validationErrors;
     }
 
+    validateCapabilities () {
+        const { layer, propertyFields } = this.getState();
+        if (propertyFields.includes(LayerComposingModel.CAPABILITIES_STYLES)) {
+            rasterStyleCapabilities(layer).forEach(field => {
+                const options = {
+                    duration: null,
+                    title: getMessage('capabilities.validate'),
+                    content: getMessage(`capabilities.rasterStyle.${field}`)
+                };
+                Messaging.warn(options);
```

Comment:
This shows a warning popup for each style that has been removed and has an override legend? Maybe we could just show one and gather up the messages in content to a list? Also renaming rasterStyleCapabilities() to something like getWarningsForRemovedStyles() or something like that might help read it better :)

Reply:
Then again it looks like it's only max two notifications so not for each style. I think that's ok.